### PR TITLE
Bump kotlin version to 1.2.60-eap-75

### PR DESCRIPTION
**Pull request by eskatos**

### CODE REVIEW

File: src/test/groovy/org/gradle/kotlin/test/RelocationTest.groovy [hunk 1]
```diff
@@ -62,13 +62,20 @@ class RelocationTest extends Specification {
...
                         classpath ('org.jetbrains.kotlin:kotlin-gradle-plugin:$kotlinPluginVersion') { force = true }
                     }
                 }
+                root.allprojects {
+                    repositories {
+                        maven {
+                            url "https://dl.bintray.com/kotlin/kotlin-eap"
+                        }
+                    }
+                }
```
Comment (lptr) [inline src/test/groovy/org/gradle/kotlin/test/RelocationTest.groovy:78]: Please extrac the URL to be used as a variable, so we can update it in one place.

Comment (lptr) [reply src/test/groovy/org/gradle/kotlin/test/RelocationTest.groovy:78]: Pinging @eskatos

**Comment by lptr** [comment]
Testing Slack integration.

**Comment by eskatos** [comment]
Superseded by #12

**Pull request closed without merging**